aspnetcore/blazor/hosting-models.md

```diff
@@ -55,7 +57,7 @@ The `blazor.webassembly.js` script is provided by the framework and handles:
 * Downloading the .NET runtime, the app, and the app's dependencies.
 * Initialization of the runtime to run the app.
 
-The Blazor WebAssembly hosting model offers several benefits:
+You should use the Blazor WebAssembly (WASM) hosting model for the following benefits:
```

Comment:
I prefer the original language here too.

Reply:
+1.  It does sound a bit clunky.